roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -172,7 +172,12 @@ rbac_enabled: "{{ 'RBAC' in authorization_modes or kubeadm_enabled }}"
 
 ## List of key=value pairs that describe feature gates for
 ## the k8s cluster.
-kube_feature_gates: ['Initializers={{ istio_enabled|string }}', 'PersistentLocalVolumes={{ local_volumes_enabled|string }}']
+kube_feature_gates: >- 
```

Comment:
Looks like you have some trailing spaces on this line that are failing the yaml linter check.